client/src/app/site/motions/services/motion-pdf.service.ts

```diff
@@ -70,9 +73,18 @@ export class MotionPdfService {
         private pollKeyVerbose: PollKeyVerbosePipe,
         private parsePollNumber: ParsePollNumberPipe,
         private motionPollService: MotionPollService,
-        private motionFormatService: MotionFormatService
+        private motionFormatService: MotionFormatService,
+        private pdfImagesService: PdfImagesService,
+        private organizationSettingsService: OrganizationSettingsService
     ) {}
 
+    /**
+     * The total width of the exported PDF in points.
+     * Is needed to calculate the width of the images in the attachment.
+     */
+    private PDF_A4_POINTS_WIDTH = 595.28;
+    private PDF_A5_POINTS_WIDTH = 419.53;
```

Comment:
Some remarks here:

- Could you please describe, how the numbers were calculated?
- Either mark them as `readonly` or (my preference) extract them as constants to the top of the file